src/plugins/foxPage/hooks/useFarmingApr.ts

```diff
@@ -1,11 +1,18 @@
+import { TransactionRequest } from '@ethersproject/abstract-provider'
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber'
```

Comment:
- We already have `BigNumber` as an alias:

https://github.com/shapeshift/web/blob/b1836d0c801e508c9312b521c6c1717e9adfdfe9/src/lib/bignumber/bignumber.ts#L4
```suggestion
import { BN } from 'lib/bignumber/bignumber'
```

- You shouldn't need `BigNumberish` which might or might not have made sense in the context of the foxfarm repo, but we already know or input types here and should be able to type `toHex` more precisely